### PR TITLE
{chem}[foss/2018b] Neper 3.4.0

### PR DESCRIPTION
**Pull request by branfosj**

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/26d0e5fd29acd9cc973bd59e5ba812ed for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3c2952b27c5bd3bd8406edf13381c122 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso-4 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/624d0adfef3da1979488219ba75aeccc for a full test report.

File: easybuild/easyconfigs/n/Neper/Neper-3.4.0_fix_missing_file_in_tests.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Add an extra test to create a file required by the tests that follow it
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/n/Neper/Neper-3.4.0_fix_missing_file_in_tests.patch]: can you please add your name and affiliation to this patch?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 3 (1 easyconfigs in total)
node2608.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/06c851fd93398ba50f541df9cec0cd27 for a full test report.

**Comment by branfosj** [comment]
replacing with #12110

**Pull request closed without merging**